homeassistant/const.py

```diff
@@ -486,11 +486,26 @@ class Platform(StrEnum):
 # Reactive power units
 POWER_VOLT_AMPERE_REACTIVE: Final = "var"
 
+
 # Energy units
-ENERGY_GIGA_JOULE: Final = "GJ"
```

Comment:
ENERGY_GIGA_JOULE was added in https://github.com/home-assistant/core/pull/80870
If this is merged before 2022.11 beta, then I think we can just remove the constant.
If it is delayed until 2022.12, then we will have to keep it in.